android/src/main/java/com/lugg/ReactNativeConfig/ReactNativeConfigPackage.java

```diff
@@ -19,11 +19,6 @@
     }
 
     @Override
-    public List<Class<? extends JavaScriptModule>> createJSModules() {
-        return Collections.emptyList();
-    }
-
-    @Override
```

Comment:
You only need to comment out the `@Override` line, then this is backwards compatible while people gradually upgrade all dependencies to get on to `0.47`.

e.g: https://github.com/wix/react-native-notifications/pull/96